contributing.md

```diff
@@ -672,9 +669,9 @@ In general, we follow [Google's style guide](https://google.github.io/styleguide
 
 ### Releasing information for members of development team
 
-[Releasing documentation](release.md) is available for the development team
-members. Includes a general overview as well as detailed information about how to
-run a new release.
+[Releasing documentation](https://github.com/ignitionrobotics/docs/blob/master/release.md)
```

Comment:
I'm not sure if this is the preferred link format; we may want to direct to ignitionrobotics.org/docs

Reply:
> I believe these web PR's haven't been merged yet. Do we need to wait for those?

arg, sorry missed the comment. We can wait for them to be merged yes.

> I'm not sure if this is the preferred link format; we may want to direct to ignitionrobotics.org/docs

found it in a different doc and though it can serve as workaround. I'm going to revert the change and we'll wait for the web-server PRs to be merged.